user/sorghum-lines-genomics-year-2.md

```diff
@@ -0,0 +1,196 @@
+# Sorghum Lines for Genomics Experiment \(Year 2\)
+
+| **accession** | [**taxid**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | [**organism common name**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | **subspecific genetic lineage rank** | **subspecific genetic lineage name** | [**ploidy**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**number of replicons**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**estimated size**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**source material identifiers**](https://www.genesys-pgr.org/) | **sample name** |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| PI 300118 | 4558 | sorghum | cultivar | PI 300118 | diploid | 10 | 366100000 | [aeb417c2-8145-4368-8bcd-6d20ecd982b8](https://purl.org/germplasm/id/aeb417c2-8145-4368-8bcd-6d20ecd982b8) | IFDN\_PCRfree\_VER886-A01.PI\_300118\_ATTACTCG\_Sorghum\_I630a\_L1 |
+| PI 300119 | 4558 | sorghum | cultivar | PI 300119 | diploid | 10 | 366100000 | [10fdc3a5-d05d-4dfd-9b06-f368d39e224a](https://purl.org/germplasm/id/10fdc3a5-d05d-4dfd-9b06-f368d39e224a) | IFDP\_PCRfree\_VER886-B01.PI\_300119\_ATTACTCG\_Sorghum\_I630a\_L4 |
```

Comment:
especially down here ... will these show up if someone copy-pastes? I think markdown only recognizes underscore for italics if there are no characters before / after the underscore, e.g. 
* `_test_` -->  _test_ 
* `t_e_st`  --> te_s_t